DotaStats/Modules/MatchesModule/View/MatchesModuleViewController.swift

```diff
@@ -18,10 +18,14 @@ protocol MatchesModuleViewOutput: AnyObject {
 }
 
 final class MatchesModuleViewController: UIViewController {
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        .lightContent
+    }
+
     // MARK: - Properties
 
     private var output: MatchesModuleViewOutput?
-    var spiner = UIActivityIndicatorView(style: .large)
+    var spinner = UIActivityIndicatorView(style: .large)
```

Comment:
Раз уж мы улучшаем код, то давай сделаем так?
```suggestion
    private lazy var spinnerView: UIActivityIndicatorView = .init(style: .large)
```